packages/cw-orch-osmosis-test-tube/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "cw-orch-osmosis-test-tube"
+version = "0.1.0"
+authors.workspace = true
+edition.workspace = true
+license.workspace = true
+repository.workspace = true
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+
+# Test Tube env deps
+osmosis-test-tube = { workspace = true }
+prost-types = { workspace = true }
+prost = { workspace = true }
+cosmwasm-std.workspace = true
+serde.workspace = true
+cw-orch = { workspace = true }
```

Comment:
Can we make this so it only depends on `cw-orch-core`. Ideally cw-orch is just a crate that ships with the default environments that you need (daemon and mock) and other environments can be plugged into it (without those envs having to depend on cw-orch itself). That way they can be independently versioned.